File: point.cpp
//
//  Point
//  Geometry
//
//  Created by Henrikh Kantuni on 12/17/16.
//  Copyright © 2016 Henrikh Kantuni. All rights reserved.
//

#include <iostream>
using namespace std;

class Point {
  private:
    double x;
    double y;
  
  public:
    Point(double x, double y) {
      this->x = x;
      this->y = y;
    }
  
    friend ostream &operator << (ostream &output, const Point &p) {
      return output << "(" << p.x << ", " << p.y << ")";
    }
};

File: point.py
""" Created by Henrikh Kantuni on 11/23/16 """


class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def __repr__(self):
        return '({0}, {1})'.format(self.x, self.y)
